README.md

```diff
@@ -1,159 +1,174 @@
-# BEM-XJST
+# bem-xjst
 
-Declarative template engine for the browser and server.
+Declarative template engine for the browser and server with regular JS syntax.
 
 [![NPM version](http://img.shields.io/npm/v/bem-xjst.svg?style=flat)](http://www.npmjs.org/package/bem-xjst)
 [![Build Status](http://img.shields.io/travis/bem/bem-xjst/master.svg)](https://travis-ci.org/bem/bem-xjst)
 [![Dependency Status](https://david-dm.org/bem/bem-xjst.svg)](https://david-dm.org/bem/bem-xjst)
 [![devDependency Status](https://david-dm.org/bem/bem-xjst/dev-status.svg)](https://david-dm.org/bem/bem-xjst#info=devDependencies)
 [![Coverage Status](https://coveralls.io/repos/github/bem/bem-xjst/badge.svg?branch=coverage-badge)](https://coveralls.io/github/bem/bem-xjst?branch=coverage-badge)
 
-[Online demo](https://bem.github.io/bem-xjst/). Twitter account: [@bemxjst](https://twitter.com/bemxjst)
+## Features
 
-## Installation
+### Templates are extensible: they can be redefined or extended
 
-Install it by [npm](https://npmjs.org): `npm install bem-xjst`.
-
-## Usage
-
-### As a node.js module
+Not only by simple redefined via new templates, but using ‘modes’ you can redefine or extend only paticular part of output. For example only tag name or tag content.
```

Comment:
You can redefine or extend just a particular part of output not only by simple redefinition via new templates but also using ‘modes’. E.g. it may be a tag name or its content.